emojirades/commands/scorekeeper_commands/history_command.py

```diff
@@ -27,7 +27,7 @@ def execute(self):
         self.logger.debug(f"Printing history: {history}")
 
         history_log = [
-            f"{index}. <@{event['user_id']}> > '{event['operation']}'"
```

Comment:
Hmmmm close! But not quite!

So slack uses the magical `<@{id}>` (where id = ABC123ABC) syntax to render the User ID => Pretty Name!

Removing the @ would then just make it print: `<ABC123ABC>` which doesn't look very pretty!

Instead we'll need to convert the `event['user_id']` => Pretty Name, there's a few examples around in the code base, for example the leaderboard code

Reply:
If you're super keen you can even go in and add a test for this!

Reply:
I was wondering why is this not using the Event class then I realized it's not slack.event lol. phew...

Reply:
btw, should be as simple and as right as `self.slack.pretty_name(event['user_id'])`?

Reply:
I was kinda hoping @Manioc909 would figure that out by themselves ;)